test/integration/ddl_sharding_key_test.lua

```diff
@@ -0,0 +1,306 @@
+local fio = require('fio')
+
+local t = require('luatest')
```

Comment:
I guess if we have space sharded by secondary index crud.select should work for such spaces because plan could check that sharding key parts are specified.

Get/update/delete shouldn't work.

Reply:
Added new tests:
```
    ddl_sharding_key[vinyl].test_select_secondary_idx
    ddl_sharding_key[vinyl].test_update_secondary_idx
    ddl_sharding_key[vinyl].test_get_secondary_idx
    ddl_sharding_key[vinyl].test_delete_secondary_idx
```